playwright.config.ts

```diff
@@ -18,7 +18,7 @@ const config: PlaywrightTestConfig = {
 	snapshotPathTemplate: '{testDir}/{testFileDir}/{testFileName}-snapshots/{arg}{ext}',
 	expect: {
 		toMatchSnapshot: {
-			maxDiffPixelRatio: 0.01
+			maxDiffPixels: 500
```

Comment:
I lowered the sensitivity of the snapshot tests because a lot of subtle UI changes were flying under the radar.